hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -112,7 +112,7 @@ public class DBScanner implements Callable<Void>, SubcommandWithParent {
 
 
   @CommandLine.ParentCommand
-  private RDBParser parent;
+  private static RDBParser parent;
```

Comment:
After passing `schemaV3` as a parameter to `displayTable`, we no longer need this change.